src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -1,8 +1,27 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.Ports;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 public class Intake extends SubsystemBase {
-    // TODO: Create empty intake methods (e.g. move up, move down, run rollers, etc.)
-    // TODO: start implementing methods
+    
+    private final CANSparkMax m_rollerMotor = new CANSparkMax(Ports.ROLLER_MOTOR_PORT, MotorType.kBrushless); //may be actually brushed not sure rn
```

Comment:
I'm pretty sure this will be brushed. In fact, you'll probably use talons for these motors, since I don't know how many sparks you have. This is good for now though.